rosbag2_tests/test/rosbag2_tests/test_rosbag2_play_end_to_end.cpp

```diff
@@ -24,25 +24,45 @@
 // rclcpp must be included before process_execution_helpers.hpp
 #include "rclcpp/rclcpp.hpp"
 
+#include "rosbag2_interfaces/srv/resume.hpp"
 #include "rosbag2_test_common/process_execution_helpers.hpp"
 #include "rosbag2_test_common/subscription_manager.hpp"
 #include "rosbag2_test_common/tested_storage_ids.hpp"
 
 #include "test_msgs/msg/arrays.hpp"
 #include "test_msgs/msg/basic_types.hpp"
-#include "test_msgs/message_fixtures.hpp"
+//  #include "test_msgs/message_fixtures.hpp"
```

Comment:
Nit: just remove this commented-out line.

Reply:
Fixed